### PR TITLE
feat: Add HTTP proxy support for OpenAI API and CLI options

### PR DESCRIPTION
**Pull request by PJovy**
- Add support for HTTP proxy configuration
- Improve CLI readability and modularization

### CODE REVIEW

**Comment by PJovy** [comment]
@yihong0618 Any comment welcomed.

File: xiao_config.json.example [hunk 1]
```diff
@@ -15,5 +15,6 @@
     "start_conversation": "开始持续对话",
     "end_conversation": "结束持续对话",
     "stream": false,
+    "proxy": ""
```
Comment (yihong0618) [inline xiao_config.json.example:18]: miss one comma

Comment (PJovy) [reply xiao_config.json.example]: Fixed.

File: xiaogpt/bot/gpt3_bot.py [hunk 2]
```diff
@@ -5,10 +5,12 @@
 
 
 class GPT3Bot:
-    def __init__(self, openai_key, api_base=None):
+    def __init__(self, openai_key, api_base=None, proxy=None):
```
Comment (frostming) [inline xiaogpt/bot/gpt3_bot.py:8]: 没找到这个选项在哪里被传入的，没有生效

**Comment by yihong0618** [comment]
@PJovy can you check?

**Pull request merged**